### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by Vichoko** [jzhang-cloud/keras-unet-vocal-separation#4]
i've just trained the model with the same dataset and hyper-parameters.
You can download it from here: [model](https://users.dcc.uchile.cl/~voyanede/more/trained_models/vocal_20.h5)
And use it in the 3rd notebook.

28s 3ms/step - loss: 0.0032 - val_loss: 0.0032

**Comment by Vichoko** [comment]
trained on ccmixter + dsd100
[model](
https://users.dcc.uchile.cl/~voyanede/more/trained_models/vocal_100_2.h5)
75s 2ms/step - loss: 0.0032 - val_loss: 0.0067
